cmake/prep/options.cmake

```diff
@@ -15,7 +15,11 @@ option(SUNSHINE_REQUIRE_TRAY "Require system tray icon. Fail the build if tray r
 
 option(SUNSHINE_SYSTEM_WAYLAND_PROTOCOLS "Use system installation of wayland-protocols rather than the submodule." OFF)
 
-option(BOOST_USE_STATIC "Use static boost libraries." ON)
+if(APPLE)
+    option(BOOST_USE_STATIC "Use static boost libraries." OFF)
+else()
+    option(BOOST_USE_STATIC "Use static boost libraries." ON)
+endif()
```

Comment:
Are you still getting issues with `BOOST_USE_STATIC=ON`? If not we should revert this.

Reply:
Without manually setting icu path, it won't build with ON, so to make easier to build locally for development, I think OFF is a good default for Clion starting setup. We could let be ON by default, but then we need to update the build docs with a step to set icu path and flags manually.

Reply:
Okay, I'm fine with it as is if we must.

Before we have different defaults though, could you investigate finding the icu path with CMake functions, such as `find_package`?

Reply:
I've tried a bit, but had no success, as icu4c is a keg-only formula, it's not widely available in the system, I plan on finding a way to do that, but I'd rather fix the build first for most users, then improve it, since I don't know how long it will take me to do it and I won't be able to build the project for a few weeks starting today.